tracekit.js

```diff
@@ -207,7 +206,7 @@ TraceKit.report = (function reportModuleWrapper() {
         var args = _slice.call(arguments, 1);
         if (lastExceptionStack) {
             if (lastException === ex) {
-                return; // already caught by an inner catch block, ignore
+                throw ex; // rethrow.
```

Comment:
Have you done any testing on this? I'm a bit hesitant about this line of code as I've seen references in other places about a stack overflow occurring.